src/__swaps__/screens/Swap/components/SwapNavbar.tsx

```diff
@@ -125,17 +121,17 @@ export function SwapNavbar() {
   );
 }
 
-export const styles = StyleSheet.create({
-  headerButton: {
-    borderRadius: 18,
-    borderWidth: THICK_BORDER_WIDTH,
-    height: 36,
-    width: 36,
-  },
-  headerTextShadow: {
-    padding: 12,
-    textShadowColor: 'rgba(0, 0, 0, 0.2)',
-    textShadowOffset: { width: 0, height: 2 },
-    textShadowRadius: 6,
-  },
-});
+// export const styles = StyleSheet.create({
+//   headerButton: {
+//     borderRadius: 18,
+//     borderWidth: THICK_BORDER_WIDTH,
+//     height: 36,
+//     width: 36,
+//   },
+//   headerTextShadow: {
+//     padding: 12,
+//     textShadowColor: 'rgba(0, 0, 0, 0.2)',
+//     textShadowOffset: { width: 0, height: 2 },
+//     textShadowRadius: 6,
+//   },
+// });
```

Comment:
remove if not needed

Reply:
This is from the settings button which is commented out above - I think we'll want to follow up sometime after release and add settings back in.